src/Appwrite/Specification/Format/OpenAPI3.php

```diff
@@ -170,6 +170,9 @@ public function parse(): array
                     'scope' => $route->getLabel('scope', ''),
                     'platforms' => $sdkPlatforms,
                     'packaging' => $route->getLabel('sdk.packaging', false),
+                    'offline-model' => $route->getLabel('sdk.offline.model', ''),
+                    'offline-key' => $route->getLabel('sdk.offline.key', ''),
+                    'offline-response-key-id' => $route->getLabel('sdk.offline.response.key.id', '$id'),
```

Comment:
I'm also open to renaming these keys

Reply:
`offline-response-key-id` could be `offline-response-key`

Reply:
@eldadfux, I've updated the label.